app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] [OPTIONAL] it would be better if you used `increment` to update the post counter, it's easier and street forward 
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```